src/app/lib/config/services/news.js

```diff
@@ -11,6 +11,39 @@ const news = {
   serviceName: 'News',
   twitterCreator: '@BBCNews',
   twitterSite: '@BBCNews',
+  translations: {
+    error: {
+      404: {
+        statusCode: '404',
+        title: 'Page can not be found',
+        message:
+          'Sorry, we’re unable to bring you the page you’re looking for. Please try:',
```

Comment:
Is this using `’` purely to avoid escaping the `'` in the JSON? 🤔

Reply:
This was copied from UX, I will check which character they want